source/extensions/extensions_build_config.bzl

```diff
@@ -71,9 +71,11 @@ EXTENSIONS = {
     "envoy.filters.http.health_check":                  "//source/extensions/filters/http/health_check:config",
     "envoy.filters.http.ip_tagging":                    "//source/extensions/filters/http/ip_tagging:config",
     "envoy.filters.http.jwt_authn":                     "//source/extensions/filters/http/jwt_authn:config",
+    # Disabled by default
```

Comment:
Can we have some test, maybe extending https://github.com/envoyproxy/envoy/blob/master/test/exe/main_common_test.cc#L459 or even just use `bazel query` that verifies that these aren't linked by default?

Reply:
https://github.com/envoyproxy/envoy/blob/master/test/exe/main_common_test.cc#L459 already covers, and it passes. Or do you want to make that test pass even with the command line flag?

Reply:
I guess my main reservation is that we probably should have an inclusion/exclusion CLI test that verifies the effects here. The reason I'm being picky is that for some sites, it's security critical to ensure that some extension are definitely excluded.

Reply:
ok added inclusion CLI test in compile time options, exclusion will be part of wasm runtime enable/disable PR.